CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
 cmake_minimum_required(VERSION 3.16.3)
-if(COMMAND CMAKE_POLICY)
-  cmake_policy(SET CMP0135 NEW)
-endif(COMMAND CMAKE_POLICY)
+cmake_policy(SET CMP0135 NEW)
```

Comment:
```suggestion
foreach(p
  CMP0135 # CMake 3.24
  )
  if(POLICY ${p})
    cmake_policy(SET ${p} NEW)
  endif()
endforeach()
```

Reply:
Specifying the version of CMake in which the policy was introduced, allows to streamline future maintenance allow to easily identify which policy toggle can be removed,

Reply:
Thank you.  Done.